### PR TITLE
Fix tickInterval

### PR DESCRIPTION
**Pull request by seanohue**
This was broken at some point (perhaps since the advent of 3.1?) by faulty logic. This addresses that and fixes it.

Co-authored by: @nelsonsbrian

### CODE REVIEW

File: src/EffectList.ts [hunk 1]
```diff
@@ -91,14 +91,18 @@ export class EffectList {
...
-					Date.now() <
-					effect.state.ticks * effect.config.tickInterval * 1000
+					sinceLastTick <
+					effect.config.tickInterval * 1000
 				) {
 					continue;
 				}
 				effect.state.lastTick = now;
-				effect.state.ticks && effect.state.ticks++;
+				if (!effect.state.ticks) {
```
Comment (nelsonsbrian) [inline src/EffectList.ts:102]: Might be redundant from:
[state.tick=0](https://github.com/Ranvier-TS/core-ts/blob/8aacd73de3e746582d95116f90f9d51e598e8129/src/Effect.ts#L176)

**Review by nelsonsbrian — APPROVED** [review]

File: src/EffectList.ts (hunk 1 above)
Comment (seanohue) [reply src/EffectList.ts:102]: I don't want to set it to zero unless it is null-ish (undefined, null, NaN), the redundancy is if it's already set to zero which is probably fine. Thinking it through, I could update the check to be `if (!effect.state.ticks && effect.state.ticks !== 0)`

Comment (nelsonsbrian) [reply src/EffectList.ts:102]: I was just pointing out that most likely the state.ticks would be 0 (unless was set differently) from the Effect constructor. Didn't know if you had seen that.

**Comment by seanohue** [comment]
That makes sense -- I went ahead and changed it to still do some defensive
coding without redundantly setting it if it's already zero.

**Pull request merged**